### ISSUE TITLE
closing a Calcdex with the Teambuilder open moves the Teambuilder to the right panel

### ISSUE THREAD

**Issue by doshidak** [doshidak/showdex#14]
If you have a single Calcdex tab and the Teambuilder open (with no other tabs, including a `ChatRoom` or `BattlesRoom`), the Teambuilder will move to the right panel once the Calcdex tab is closed.

This is caused by the overwritten close tab handler for Calcdex tabs, which causes the Calcdex to visually close, but is actually hidden in order to preserve the state (this was the easiest and quickest way for me to implement re-opening Calcdex tabs before pushing out v1.0.1).

In that close tab handler, I forgot to check if the room we're refocusing into is a side room by reading the room's `isSideRoom` property. The Teambuilder tab is **not** a side room, so its `isSideRoom` property would be `false`. Filtering would remove the Teambuilder from the list of focusable side rooms, preventing it from opening on the right panel.

(There is logic already in place to hide the right panel if there are no more focusable side rooms. Clicking on the "+" tab would show the hidden right panel again.)

In other words, this should be a straightforward fix.

**Comment by doshidak** [comment]
Fixed this.

Will close this via reference on the next PR.
